assets/scammer/camelshouse.json

```diff
@@ -0,0 +1,44 @@
+{
+    "metadata": {
+        "label": "scammer",
```

Comment:
I don't think this is a great label here. I'd call it `scammer_camelshouse`. Otherwise some scammer activities may be merged together.